BlueBrain/nexus-web
# Fix displaying of polylines when there are multiple previous steps

Fixes BlueBrain/nexus#2496

A step with a single previous would render the connecting polyline correctly however no connecting polyline would be displayed when there are multiple previous steps due to an issue with the polyline id not being set correctly. Updated to set id of polyline correctly when rendering array of prev steps.

![2496_workflow_step_links](https://user-images.githubusercontent.com/11296166/122801456-6bdcfd80-d2c4-11eb-9007-4f88a7149613.gif)

### Testing
1. Create several workflow steps
2. Click into one of the steps
3. Open the step info and add multiple previous steps
4. Navigate back to the view with the steps
5. Observe the previous steps connecting arrows

### Expected Result
Connecting arrows for each of the previous steps shown on the canvas linked to the step we specified.
